src/main/java/InboxPage.java

```diff
@@ -0,0 +1,21 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+class InboxPage extends Page{
+    private final static String TITLE = "Входящие - Почта Mail.Ru";
+    private static final By LOGOUT_LINK = By.id("PH_logoutLink");
+
+    InboxPage() {
+        super(TITLE);
+    }
+
+    LoginPage logout() {
+        getLogoutLink().click();
+        return new LoginPage();
+    }
+
+    WebElement getLogoutLink() {
```

Comment:
It is strange that you are returning WebElement and method is not private. Page Object should encapsulate everything connected to WebElement. Please update